.github/workflows/check_style.yaml

```diff
@@ -0,0 +1,24 @@
+name: Style Checks
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - '*'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - name: Install requirements
+      run: pip install -r requirements.txt
+    - name: Check Code format
+      working-directory: Tools
```

Comment:
This directory doesn't exist ?

Reply:
Ah, wait you added it in the PR :joy:

Reply:
yeah there was a typo i overlooked in the naming of the script....